packages/nimble-components/src/tooltip/specs/README.md

```diff
@@ -14,6 +14,102 @@ The nimble-tooltip project will first be implemented as a prototype, open issues
 
 ---
 
+
+## Potential approaches
+
+Below we enumerate several different design approaches from which we may choose one or more as our ideal API surface.
+
+### Prior art
+- [Angular Material Tooltip](https://material.angular.io/components/tooltip/overview) - Uses Angular specific functionality (Directive)
+- [Carbon Design System](https://carbondesignsystem.com/components/tooltip/usage#live-demo) - Tooltip component takes anchor element as content.
+- [Atlassian Design System](https://atlassian.design/components/tooltip/examples) - Tooltip component takes anchor element as content
+- [Adobe Spectrum (React)](https://react-spectrum.adobe.com/react-spectrum/Tooltip.html) - Separate wrapper component around tooltip and anchor elements.
+- [FAST](https://explore.fast.design/components/fast-tooltip) - Tooltip associated with anchor element by id.
+
+### Associate by `id`
+This is the FAST approach, and the current implementation. The tooltip has an `anchor` attribute which takes the `id` of the anchor element.
+```html
+<nimble-tooltip anchor="previewButton">...</nimble-tooltip>
+<nimble-button id="previewButton">Preview</nimble-button>
+```
+
+**Pros**
+- No additional implementation cost
+- Familiar pattern, e.g. `<label>` element with `for` attribute, `aria-labelledby` attribute, `aria-describedby` attribute
+- Reasonable DOM structure, i.e. tooltip sibling of anchor element
+
+**Cons**
+- Annoying to have to give anchor element an `id`
+- <span style="color:red">**Not conducive to use in shared components because `id` values must be unique**</span>
```

Comment:
I'll mention it in one spot but please remove the html style formatting with `<span style="color:red">`. It doesn't render on GitHub and makes the markdown harder to read.